dynoid/dynoid.go

```diff
@@ -56,31 +241,60 @@ func (v *Verifier) Verify(ctx context.Context, rawIDToken string) (*oidc.IDToken
 	return verifier.Verify(ctx, rawIDToken)
 }
 
+// VerifyHeroku verifies the token and parses the returned issuer and subject
+// according to Heroku expected values
+func (v *Verifier) VerifyHeroku(ctx context.Context, rawIDToken string) (*Token, error) {
```

Comment:
Considering this is our own package, I think we should merge `Verify` and `VerifyHeroku`.

This seems simpler to maintain so we don't risk someone else starts using it for a use case it wasn't designed for.

Reply:
Fixed in 91aad96